helix-core/src/main/java/org/apache/helix/controller/stages/AttributeName.java

```diff
@@ -40,5 +40,6 @@
   PipelineType,
   LastRebalanceFinishTimeStamp,
   ControllerDataProvider,
-  STATEFUL_REBALANCER
+  STATEFUL_REBALANCER,
+  CONTROLLER_LEADER_SESSION
```

Comment:
This can be more generic. Just call it event_session?
As you can imagine, this can be used in multiple ways.